celltype_annotation/analysis/copykat-rms-exploration.Rmd

```diff
@@ -0,0 +1,433 @@
+---
+title: "CopyKAT: Rhabdomyosarcoma cell type tumor vs normal inference for `r {params$library_id}`"
+output: 
+  html_notebook:
+  toc: true
+toc_float: true
+params:
+  s3_singler_data_dir: "s3://nextflow-ccdl-results/scpca/processed/results/SCPCP000005" 
+  s3_submitter_data_dir: "s3://sc-data-integration/scpca/celltype_sce"
+  local_data_dir: "../data"
+  library_id: "SCPCL000488"
+  sample_id: "SCPCS000262"
+  test: FALSE
+  threads: 4
+---
+  
+This notebook explores performance of the [`copyKat`](https://github.com/navinlabcode/copykat) package for calling tumor vs. normal cells in pediatric cancer scRNA-seq data.
+Specifically, we look at a library from the `SCPCP000005` (RMS) project, where the submitter has annotated cell types which we can consider "ground truth" for the purposes of this exploration.
+We'll assess agreement between the submitter tumor/normal classification and what `copyKat` infers as tumor/normal, across some different `copyKat` parameter values.
+
+## Background: `copyKat` method
+
+Reference: https://doi.org/10.1038/s41587-020-00795-2
+
+To begin, here's a brief overview of the `copyKat` approach, parly quoting the paper throughout.
+See Figure 1 in the paper for a method diagram:
+
+* `copyKat` takes the _raw expression matrix_ and stabilizes variance, smooths the outliers
+* A subset of cells is then identified to serve as a diploid baseline for subsequent identification of diploid/aneuploid
```

Comment:
Is this information returned at all? Can you go back and see which cells are being classified as the "confident diploid"?

Reply:
I wish 🙁 
https://github.com/AlexsLemonade/sc-data-integration/pull/230#issuecomment-1686712505